Makefile

```diff
@@ -154,7 +154,7 @@ nuget-install:
 .PHONY: template-install
 template-install:
 	@printf "Installing ${GREEN}C# Template packages${DEFAULT}\n"
-	@-dotnet new uninstall $(THISDIR)/source/templates/csharp/
+	@-dotnet new --uninstall $(THISDIR)/source/templates/csharp/
```

Comment:
When running dotnet new --uninstall on my system, I get this: `Warning: use of 'dotnet new --uninstall' is deprecated. Use 'dotnet new uninstall' instead.`

Reply:
@GreenData17 what version of dotnet are you using? You might have an older dotnet version where using only `uninstall` isn't possible

Reply:
I am using the dotnet-sdk-6.0 from the AUR. (its the version mentioned in the documentation)

Reply:
weird I am also using the same one, you should leave it as is, what error were you getting?

Reply:
Those three:

![Screenshot from 2024-07-10 00-50-48](https://github.com/CosmosOS/Cosmos/assets/57272730/ee6503fd-74b0-43d6-89b6-8595379b795b)

After adding the double dashes, all of them disappeared.

Reply:
Some more information for finding the problem: 
I use arch and install applications mostly via yay. 
To install the dotnet sdk I executed `yay -S dotnet-sdk-6.0`. 
When executing `dotnet --version` I get the version "6.0.131". 
Hope this helps.

Reply:
For dotnet6 and older, `--uninstall` should be used according to the docs. https://learn.microsoft.com/en-us/dotnet/core/tools/dotnet-new#description The change to subcommands was introduced in .net 7.

Reply:
Ok, so for users like me, who only have the sdk 6.0 installed the command should have `--uninstall`.
And the warning probably only appears for users who have newer versions or multiple sdk's installed.
@selkij how is it for you, do you have a newer version installed? and what do you think should happen with this problem?

Reply:
I also use the net6.0 sdk